_ml-commons-plugin/api.md

```diff
@@ -729,8 +732,147 @@ GET /_plugins/_ml/stats
 }
 ```
 
+## Execute
+
+Some algorithms, such as [Localization]({{site.url}}{{site.baseurl}}/ml-commons-plugin/algorithms#localization), don't require trained models. You can run no-model-based algorithms using the `execute` API.
 
+```json
+POST _plugins/_ml/_execute/<algorithm_name>
+```
 
+### Example: Execute localization 
 
+The following example uses the localization algorithm to find subset-level information for aggregate data (for example, aggregated over time) that demonstrates the activity of interest, such as spikes, drops, changes, or anomalies.
```

Comment:
"Localization"  for Localization algorithm should be capitalized here as well, I believe. 
```suggestion
The following example uses the Localization algorithm to find subset-level information for aggregate data (for example, aggregated over time) that demonstrates the activity of interest, such as spikes, drops, changes, or anomalies.
```